### PR TITLE
added example for iris web framework

### PR DESCRIPTION
**Pull request by mehulgohil**
### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### 🔧 Changes
This PR adds an example for the iris web framework

### 📚 References
[Iris](https://github.com/kataras/iris)

### CODE REVIEW

File: examples/iris-example/go.mod [hunk 1]
```diff
@@ -0,0 +1,54 @@
+module example.com/iris
+
+go 1.20
+
+require (
+	github.com/auth0/go-jwt-middleware/v2 v2.1.0
+	github.com/kataras/iris/v12 v12.2.0
+)
+
```
Comment (ewanharris) [inline examples/iris-example/go.mod:9]: I think similar to the other examples we should add a replace here so the local version of this repo gets used


```suggestion

replace github.com/auth0/go-jwt-middleware/v2 => ./../../

```

**Review by ewanharris — CHANGES_REQUESTED** [review]
Thanks for the PR @mehulgohil! I've just left a couple comments around consistency with other examples, but other than that this looks good!

File: examples/iris-example/main.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+		if len(customClaims.Username) == 0 {
+			ctx.StopWithJSON(
+				http.StatusBadRequest,
+				map[string]string{"message": "Username in JWT claims was empty."},
+			)
+			return
+		}
+		log.Println(claims)
+		ctx.JSON(http.StatusOK)
```
[the remark this answers is not in the document]
Comment (mehulgohil) [reply examples/iris-example/main.go]: Resolved

**Comment by mehulgohil** [comment]
Hi @ewanharris 
Resolved both the comments

**Review by ewanharris — APPROVED** [review]
Thanks @mehulgohil

**Pull request merged**